rbac/management/principal/model.py

```diff
@@ -60,4 +60,5 @@ class Meta:
         ordering = ["username"]
         constraints = [
             models.UniqueConstraint(fields=["username", "tenant"], name="unique principal username per tenant"),
+            models.UniqueConstraint(fields=["user_id"], name="unique user id for the principal"),
```

Comment:
@MikelAlejoBR do we know whether user_id has to be unique globally or per tenant ?